Description: Building documentation with Sphinx uses intersphinx by
 default, which downloads data from the internet. This isn't allowed
 during Debian binary builds.
Author: Paul Gevers <elbrus@debian.org>
Forwarded: not-needed
Index: nbconvert/docs/source/conf.py
===================================================================
--- nbconvert.orig/docs/source/conf.py
+++ nbconvert/docs/source/conf.py
@@ -312,7 +312,9 @@ texinfo_documents = [
 
 # Example configuration for intersphinx: refer to the Python standard library.
 intersphinx_mapping = {
-    "python": ("https://docs.python.org/3.6", None),
-    "jinja": ("http://jinja.pocoo.org/docs", None),
-    "nbformat": ("https://nbformat.readthedocs.io/en/latest", None),
+# Disabled in Debian because this fetches data during build which isn't allowed.
+# See https://bugs.debian.org/1023040
+#    "python": ("https://docs.python.org/3.6", None),
+#    "jinja": ("http://jinja.pocoo.org/docs", None),
+#    "nbformat": ("https://nbformat.readthedocs.io/en/latest", None),
 }
